Makefile

```diff
@@ -233,6 +233,28 @@ deploy-test: _verify_minikube generate-keys
 	export ENABLE_AUTH="True" && export DUMMY_IGNITION="True" && \
 	$(MAKE) _update-minikube deploy-wiremock deploy-all
 
+generate-onprem-environment:
+	echo "POSTGRESQL_DATABASE=installer" > onprem-environment
+	echo "POSTGRESQL_PASSWORD=admin" >> onprem-environment
+	echo "POSTGRESQL_USER=admin" >> onprem-environment
+	echo "DB_HOST=127.0.0.1" >> onprem-environment
+	echo "DB_PORT=5432" >> onprem-environment
+	echo "DB_USER=admin" >> onprem-environment
+	echo "DB_PASS=admin" >> onprem-environment
+	echo "DB_NAME=installer" >> onprem-environment
+	echo "SERVICE_BASE_URL=http://127.0.0.1:8090" >> onprem-environment
+	echo "DEPLOY_TARGET=onprem" >> onprem-environment
+	echo "DUMMY_IGNITION=false" >> onprem-environment
+	echo "OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> onprem-environment
+
+generate-onprem-iso-ignition:
+	sed -i "s|OPENSHIFT_INSTALL_RELEASE_IMAGE=.*|OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}|" ./config/onprem-iso-fcc.yaml
+	podman run --rm -v ./config/onprem-iso-fcc.yaml:/config.fcc:z quay.io/coreos/fcct:release --pretty --strict /config.fcc > ./config/onprem-iso-config.ign
+
+verify-latest-onprem-config: generate-onprem-environment generate-onprem-iso-ignition
```

Comment:
I think this check should be performed as part of PR verification. Could we add this to the Build stage in the Jenkinsfile?

Reply:
Done

Reply:
Please add this as part of the `ci-lint` target

Reply:
You mean like this? 
```
lint: 
	golangci-lint run -v
        $(MAKE) verify-latest-onprem-config
```

Reply:
Yes. but for this one
```make
ci-lint:
	${ROOT_DIR}/tools/check-commits.sh
        $(MAKE) verify-latest-onprem-config
```